packages/api/src/constants.js

```diff
@@ -1,2 +1,12 @@
 export const GATEWAY = new URL('https://ipfs.io')
 export const JWT_ISSUER = 'web3-storage'
+
+export const cluster = {
+  apiUrl: typeof CLUSTER_API_URL !== 'undefined'
+    ? CLUSTER_API_URL
+    : '',
+  basicAuthToken:
+    typeof CLUSTER_BASIC_AUTH_TOKEN !== 'undefined'
+      ? CLUSTER_BASIC_AUTH_TOKEN
+      : '',
```

Comment:
We won't be able to do this when we switch to esm - can you create the cluster client in the `envAll.js`?